app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    if user.present?
+      can %i(create read update destroy), Journal
+      can %i(create read update destroy), JournalEntry
```

Comment:
Can you try this out?
```suggestion
      can %i(create read update destroy), [Journal, JournalEntry]
```

Reply:
sorry, I actually implemented it and changed it again due to some errors